app/services/ScratchService.scala

```diff
@@ -28,11 +28,11 @@ import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 
 @Singleton
-class ScratchService @Inject() (repository: ScratchRepository, pageBuilder: PageBuilder) {
+class ScratchService @Inject() (repository: ScratchRepository, pageBuilder: PageBuilder, securedProcessBuilder: SecuredProcessBuilder) {
   val logger = Logger(getClass)
 
   def save(process: JsObject): Future[RequestOutcome[UUID]] =
-    guidancePages(pageBuilder, process).fold(
+    guidancePages(pageBuilder, securedProcessBuilder: SecuredProcessBuilder, process).fold(
```

Comment:
I don't think you require the type definition for SecuredProcessBuilder.

Reply:
cut and paste ;) , removed